### PR TITLE
223 Dev Edition User Types Survey

### PR DESCRIPTION
**Pull request by rayborn**
Please review and test.  **We need to wait for a cue from Bradley to launch**

### CODE REVIEW

File: src/recipes/heartbeat-by-user-first-impression/index.js [hunk 1]
```diff
@@ -245,23 +247,22 @@ let run = function (state, extras) {
     }
   };
 
-  let breaks = [1.0];
-
 
+  let channel_breaks = {'aurora':[.3]};
```
Comment (gregglind) [inline src/recipes/heartbeat-by-user-first-impression/index.js]: why aren't you just taking everyone from aurora?  There is no other live survey.

Comment (rayborn) [inline src/recipes/heartbeat-by-user-first-impression/index.js]: We only need a few hundred users, and I was thinking of running for a full week to keep weekday/weekend users.  If oversampling isn't an issue then I could just do 1.0.

Also, this doesn't seem to be working in practice.  I seem to be getting the survey 100% on Aurora.

Comment (gregglind) [inline src/recipes/heartbeat-by-user-first-impression/index.js]: I will just up you to that, and rebase and push :)

**Comment by gregglind** [comment]
Deprecated by issue #227.  Closing.

**Pull request closed without merging**